add section splitting for BE_ZivilStraf

Skatinger opened the pull request:
# What & Why
added section splitting for BE_ZivilStraf spider.

# Results
![image](https://user-images.githubusercontent.com/26817152/145447759-aea24faa-e65b-4193-8d08-3a6c257cf4b6.png)

The german parser sadly missed a lot of rulings as the section cannot be distinguished by regex alone.
Example: The first match is within a summary in the considerations section, the other is the header of the ruling:
![image](https://user-images.githubusercontent.com/26817152/145447952-24f88d99-0c79-4ac7-8e2f-08272a14ebdd.png)

There is no easy way to fix this, and none of the possibilites are really worth the effort I'm afraid.

The spider could still be improved, but for the last ~15% of missed sections nearly all are edge-cases with typos or do not have a clean structure.

# Remarks
- This spider does not have a facts section. Facts and considerations are concatenated. I filled them into the considerations section.

# Important
This PR depends on https://github.com/JoelNiklaus/SwissCourtRulingCorpus/pull/26. It will not work otherwise, as it uses the new functionality added in the mentioned PR. Do not merge until the other PR has been accepted.

JoelNiklaus commented:
Looks good. Could you please also note down the reasons for not implementing further possibilities in comments somewhere in the code or in the google sheets overview document?

Skatinger commented:
> Looks good. Could you please also note down the reasons for not implementing further possibilities in comments somewhere in the code or in the google sheets overview document?

Just realized I made a horrible mistake. Turns out I could not match rulings in about 25%, because there was just a single sentence for it represented by the last paragraph in the considerations section which I just dismissed.
Examples:

- Aus den Darlegungen erhellt, dass die Konkursandrohung in der Betreibung Nr. 123
        des Betreibungs- und Konkursamtes B, Dienststelle P., nicht nichtig ist.
 - Nach dem Gesagten ist auf die Beschwerde nicht einzutreten.
 - Vor diesem Hintergrund ist das Vorgehen der Dienststelle unter den dargelegten
        Umständen nicht zu beanstanden und die vorliegende Beschwerde abzuweisen.

![image](https://user-images.githubusercontent.com/26817152/145549806-9ca04a4b-536b-4a1e-b0ed-9067d2ea4962.png)

My implementation doesn't match a lot of dismissed cases, so I will definitely try to fix this first before you can merge.

Skatinger commented:
Went above and beyond to try to extract some more rulings. Not sure if the effort was worth it in comparison to the time invested, but at least it did bring some better accuracy. Rulings are now about 14% better than before. I also added a lot of info to the spider, I really like the idea as it will help if someone in the future wants to improve the spider.

![image](https://user-images.githubusercontent.com/26817152/145711356-3664abf3-ae40-4144-be77-6137a88fb4d9.png)


Secondly I fixed the `log_coverage_from_json` for cases where decision do not have a specific section. For this spider this was the facts section, which was NaN in all columns, but the sum() function of pandas would count those as values, giving columns with all NaN values a perfect score. Now it correctly returns 0% (see screenshot above).

JoelNiklaus commented:
Thanks for the update. Looks good.
